### PR TITLE
reuse Ansible playbooks to initialize the database

### PR DESCRIPTION
**Pull request by ddragosd**
This pull request uses a docker container with `ansible` to execute 2 ansible playbooks: 
1. `initdb.yml`
2. `wipe.yml` 

These 2 playbooks initialize the database.

Another option would be to create an OpenWhisk docker container that initializes itself but until we have that I'm thinking to use this simplified script.

### CODE REVIEW

File: docker-compose/Makefile [hunk 1]
```diff
@@ -92,35 +84,14 @@ init-couchdb:
...
-	echo db.username=$(OPEN_WHISK_DB_USERNAME) >> $(PROJECT_HOME)/whisk.properties
-	echo db.password=$(OPEN_WHISK_DB_PASSWORD) >> $(PROJECT_HOME)/whisk.properties
-	echo db.whisk.actions=$(OPEN_WHISK_DB_ACTIONS) >> $(PROJECT_HOME)/whisk.properties
-	echo db.whisk.apigw=$(OPEN_WHISK_DB_GW) >> $(PROJECT_HOME)/whisk.properties
-	cd $(PROJECT_HOME)/ && tools/db/wipeTransientDBs.sh
-	rm $(PROJECT_HOME)/whisk.properties
+	# make sure the src files are in a shared folder for docker
+	rm -rf ~/tmp/openwhisk/src
+	ln -s $(PROJECT_HOME) ~/tmp/openwhisk/src
+	docker run --rm -v ~/tmp/openwhisk/src:/openwhisk -w /openwhisk/ansible \
```
Comment (rabbah) [inline docker-compose/Makefile]: nice

**Comment by ddragosd** [comment]
@rabbah  @markusthoemmes may I get your help with this PR ?
 
I'm thinking to use a similar strategy to build a default Openwhisk DB image that will be used to startup faster, not just for `docker-compose` but also for `mesos`. Then we can replace this script but the concept will be the same. This idea came up from [this discussion](https://lists.apache.org/thread.html/76e64c8c658d2c0f201ee1b9412d4658e44acc7d471d066669f25abb@%3Cdev.openwhisk.apache.org%3E) on the Openwhisk.

**Comment by markusthoemmes** [comment]
@ddragosd do you aim to have a couchdb image which already has the databases set up?

**Comment by ddragosd** [comment]
@markusthoemmes that's right. I'm also working on that. see this commit: https://github.com/openwhisk/openwhisk/commit/fc1df9f236cddd64f782497d41de806de8e7f5cf . I'll probably turn it into a PR once I test it successfully in a Mesos/DCOS environment. I was able to test it already with docker-compose and it's working fine.

**Comment by ddragosd** [comment]
I'm cross referencing this PR in OpenWhisk repo that would simplify these changes more by reusing the couchdb container.

**Comment by style95** [comment]
I am not sure this is the right place to write, but I hope upper case db name issue reported in [this issue](https://github.com/openwhisk/openwhisk/issues/1431) is also taken account.

**Comment by ddragosd** [comment]
As discussed in https://github.com/openwhisk/openwhisk/pull/1928 we're going to build the couchDB container as a separate project inside of this repository. 

I'm going to create a separate PR for building the couchDB container which will eventually be also pushed to Docker Hub.

Until then we need to merge this PR due to https://github.com/openwhisk/openwhisk/pull/1971

**Pull request merged**